app/models/order.rb

```diff
@@ -1,6 +1,6 @@
 class Order <ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
-  enum status: %w(pending packaged shipped cancelled)
+  enum status: %w(packaged pending shipped cancelled)
```

Comment:
Brand new orders are now defaulting to `packaged`, so we'll need a migration to change the default status of the orders table

Reply:
Great catch, and done!